.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "harfbuzz"]
 	path = harfbuzz
 	url = https://github.com/harfbuzz/harfbuzz
+	branch = 2.6.8
```

Comment:
I don’t think this is what I meant, sorry for confusing you. I don’t think any action was needed here, what you had was fine.

Reply:
no problem, gone